src/coreclr/inc/nsutilpriv.h

```diff
@@ -160,7 +160,7 @@ void MakePath(                          // throws on out of memory
     const SString &ssName);             // Final part of name.
 
 //*****************************************************************************
-// Concatinate type names to assembly names
+// Concatenate type names to assembly names
 //*****************************************************************************
 static
 bool MakeAssemblyQualifiedName(                                  // true if ok, false if out of memory
```

Comment:
MakeAssemblyQualifiedName methods look like dead code. Could you please delete them instead?